--- tau-2.28/installtau	2017-11-06 18:05:39.000000000 -0800
+++ patch-files/installtau	2018-11-29 16:51:18.596933071 -0800
@@ -348,37 +348,37 @@
 
     # PDT
     echo "Configuring with ./configure $common $pdtopt $mpiopts"
-    ./configure $common $pdtopt $mpiopts <wishver
+    ./configure $common $pdtopt $mpiopts -bfd=/opt/gnu <wishver
     make clean; make $parallel install $TAU_MAKE_OPT
 
     # configure with perflib using perfopts
     if [ $perf = yes ] ; then
 	echo "Configuring with ./configure $common $pdtopt $mpiopts $perfopts"
-	./configure $common $pdtopt $mpiopts $perfopts <wishver
+	./configure $common $pdtopt $mpiopts $perfopts -bfd=/opt/gnu <wishver
 	make clean; make $parallel install $TAU_MAKE_OPT
     fi
 
     # -DEPTHLIMIT
     echo "Configuring with ./configure $common $pdtopt $mpiopts -DEPTHLIMIT"
-    ./configure $common $pdtopt $mpiopts -DEPTHLIMIT <wishver
+    ./configure $common $pdtopt $mpiopts -DEPTHLIMIT -bfd=/opt/gnu <wishver
     make clean; make $parallel install $TAU_MAKE_OPT
 
     # -PROFILEPARAM
     echo "Configuring with ./configure $common $pdtopt $mpiopts -PROFILEPARAM"
-    ./configure $common $pdtopt $mpiopts -PROFILEPARAM <wishver
+    ./configure $common $pdtopt $mpiopts -PROFILEPARAM -bfd=/opt/gnu <wishver
     make clean; make $parallel install $TAU_MAKE_OPT
 
     # Epilog
     if [ $epilog = yes ] ; then
 	echo "Configuring with ./configure $common $pdtopt $mpiopts -TRACE $vtfopt $otfopt $epilogopt $papiopt"
-	./configure $common $pdtopt $mpiopts -TRACE $vtfopt $otfopt $epilogopt $papiopt <wishver
+	./configure $common $pdtopt $mpiopts -TRACE $vtfopt $otfopt $epilogopt $papiopt -bfd=/opt/gnu <wishver 
 	make clean; make $parallel install $TAU_MAKE_OPT
     fi
 
     # VampirTrace
     if [ $vampirtrace = yes ] ; then
 	echo "Configuring with ./configure $common $pdtopt $mpiopts -TRACE $vtfopt $otfopt $vptopt $papiopt"
-	./configure $common $pdtopt $mpiopts -TRACE $vtfopt $otfopt $vptopt $papiopt <wishver
+	./configure $common $pdtopt $mpiopts -TRACE $vtfopt $otfopt $vptopt $papiopt -bfd=/opt/gnu <wishver
 	make clean; make $parallel install $TAU_MAKE_OPT
     fi
     
@@ -388,19 +388,19 @@
       ./configure $common $pdtopt $mpiopts $papiopt <wishver
       make clean; make $parallel install $TAU_MAKE_OPT
       echo "Configuring with ./configure $common $pdtopt $mpiopts $papiopt -PROFILEPHASE"
-      ./configure $common $pdtopt $mpiopts $papiopt -PROFILEPHASE <wishver
+      ./configure $common $pdtopt $mpiopts $papiopt -PROFILEPHASE -bfd=/opt/gnu <wishver
       make clean; make $parallel install $TAU_MAKE_OPT
     fi
 
     if [ $opari = yes ] ; then
 	echo "Configuring with ./configure $common $pdtopt $mpiopts -opari"
-	./configure $common $pdtopt $mpiopts -opari -openmp <wishver
+	./configure $common $pdtopt $mpiopts -opari -openmp -bfd=/opt/gnu <wishver
 	make clean; make $parallel install $TAU_MAKE_OPT
     fi 
 
     if [ $opari = yes -a $papi = yes ] ; then
 	echo "Configuring with ./configure $common $pdtopt $mpiopts -opari $papiopt "
-	./configure $common $pdtopt $mpiopts -opari -openmp $papiopt <wishver
+	./configure $common $pdtopt $mpiopts -opari -openmp $papiopt -bfd=/opt/gnu  <wishver
 	make clean; make $parallel install $TAU_MAKE_OPT
     fi 
 
@@ -410,12 +410,12 @@
 
 if [ $opari = yes ] ; then
     echo "Configuring with $common $pdtopt -opari -openmp "
-    ./configure $common $pdtopt -opari -openmp <wishver
+    ./configure $common $pdtopt -opari -openmp -bfd=/opt/gnu <wishver
     make clean; make $parallel install $TAU_MAKE_OPT
     
     if [ $papi = yes ] ; then 
 	echo "Configuring with $common $pdtopt -opari -openmp $papiopt  "
-	./configure $common $pdtopt -opari -openmp $papiopt <wishver
+	./configure $common $pdtopt -opari -openmp $papiopt -bfd=/opt/gnu <wishver
 	make clean; make $parallel install $TAU_MAKE_OPT
     fi
 fi 
@@ -423,7 +423,7 @@
 if [ $arch = no -o $archdir != bgl -a $archdir != xt3 ] ; then
       # Just Pthread
     echo "Configuring with $common $pdtopt -pthread "
-    ./configure $common $pdtopt -pthread <wishver
+    ./configure $common $pdtopt -pthread -bfd=/opt/gnu <wishver
     make clean; make $parallel install $TAU_MAKE_OPT
 fi
 
@@ -431,27 +431,27 @@
     # PAPI with PTHREAD 
     if [ $arch = no -o $archdir != bgl -a $archdir != xt3 ] ; then
 	echo "Configuring with -pthread $papiopt  "
-	./configure $common $pdtopt -pthread $papiopt  <wishver
+	./configure $common $pdtopt -pthread $papiopt  -bfd=/opt/gnu <wishver
 	make clean; make $parallel install $TAU_MAKE_OPT
     fi
     
     # PAPI with -pdt 
     if [ $pdt = yes ] ; then
 	echo "Configuring with $common $pdtopt $papiopt "
-	./configure $common $pdtopt $papiopt <wishver
+	./configure $common $pdtopt $papiopt -bfd=/opt/gnu <wishver
 	make clean; make $parallel install $TAU_MAKE_OPT
     fi
 fi
 
 if [ $dyninst = yes ] ; then
     echo "Configuring with $common -dyninst=$dyninstdir "
-    ./configure $common -dyninst=$dyninstdir < wishver
+    ./configure $common -dyninst=$dyninstdir -bfd=/opt/gnu < wishver
     make clean; make $parallel install $TAU_MAKE_OPT
 fi
 
 # Serial library
 echo "Configuring with $common $pdtopt "
-./configure $common $pdtopt $vtfopt $otfopt <wishver
+./configure $common $pdtopt $vtfopt $otfopt -bfd=/opt/gnu <wishver
 make clean; make $parallel install $TAU_MAKE_OPT
 
 # Set file permissions
