--- tau_instrumentor.cpp	2018-05-29 10:28:07.712574868 -0700
+++ patch-files/tau_instrumentor.cpp	2018-05-29 10:29:21.108577123 -0700
@@ -2699,7 +2699,7 @@
 #endif /* DEBUG */
     isfree = isFreeFormat(inbuf);
     do {
-      if (istr.getline(allocstmt, INBUF_SIZE) == NULL) {
+      if ( !istr.getline(allocstmt, INBUF_SIZE) ) {
         perror("ERROR in reading file: looking for ) for continuation line instrumentation of alloc/dealloc");
         exit(1);
       }
@@ -2807,7 +2807,7 @@
 #endif /* DEBUG */
     isfree = isFreeFormat(inbuf);
     do {
-       if (istr.getline(deallocstmt, INBUF_SIZE) == NULL)
+       if ( !istr.getline(deallocstmt, INBUF_SIZE) )
        {
          perror("ERROR in reading file: looking for ) for continuation line instrumentation of alloc/dealloc");
          exit(1);
@@ -2920,7 +2920,7 @@
     printf("Contination line: %s\n", inbuf);
 #endif /* DEBUG */
     isfree = isFreeFormat(inbuf);
-    if (istr.getline(iostmt, INBUF_SIZE) == NULL)
+    if ( !istr.getline(iostmt, INBUF_SIZE) )
     {
       perror("ERROR in reading file: looking for continuation line instrumentation of io");
       exit(1);
