--- TauMpi.c	2018-05-29 12:06:27.630756101 -0700
+++ patch-files/TauMpi.c	2018-05-29 18:06:20.321419382 -0700
@@ -1752,7 +1752,7 @@
 
 
 int  MPI_Address( location, address )
-TAU_MPICH3_CONST void * location;
+void * location;
 MPI_Aint * address;
 {
   int  returnVal;
@@ -2762,8 +2762,8 @@
 
 int  MPI_Type_hindexed( count, blocklens, indices, old_type, newtype )
 int count;
-TAU_MPICH3_CONST int * blocklens;
-TAU_MPICH3_CONST MPI_Aint * indices;
+int * blocklens;
+MPI_Aint * indices;
 MPI_Datatype old_type;
 MPI_Datatype * newtype;
 {
@@ -2851,9 +2851,9 @@
 
 int  MPI_Type_struct( count, blocklens, indices, old_types, newtype )
 int count;
-TAU_MPICH3_CONST int * blocklens;
-TAU_MPICH3_CONST MPI_Aint * indices;
-TAU_MPICH3_CONST MPI_Datatype * old_types;
+int * blocklens;
+MPI_Aint * indices;
+MPI_Datatype * old_types;
 MPI_Datatype * newtype;
 {
   int  returnVal;
