--- papi-5.6.0/src/ctests/Makefile.recipies	2017-12-19 09:14:24.000000000 -0800
+++ patch-files/Makefile.recipies	2018-11-30 12:10:35.769419108 -0800
@@ -31,13 +31,6 @@
 RANGE	= data_range
 BROKEN	= pernode val_omp
 API = api
-ifneq ($(MPICC),)
-ALL	= $(PTHREADS) $(SERIAL) $(FORKEXEC) $(OVERFLOW) $(PROFILE) $(MPI) $(MPX) $(MPXPTHR) $(OMP) $(SMP) $(SHMEM)\
-	  $(SHARED) $(EAR) $(RANGE) $(P4_TEST) $(ATTACH) $(API)
-else
-ALL	= $(PTHREADS) $(SERIAL) $(FORKEXEC) $(OVERFLOW) $(PROFILE) $(MPX) $(MPXPTHR) $(OMP) $(SMP) $(SHMEM)\
-	  $(SHARED) $(EAR) $(RANGE) $(P4_TEST) $(ATTACH) $(API)
-endif
 
 DEFAULT = papi_api serial forkexec_tests overflow_tests profile_tests attach multiplex_and_pthreads shared
 
